packages/near-api-js/src/account_multisig.ts

```diff
@@ -88,19 +98,14 @@ export class AccountMultisig extends Account {
             throw e;
         }
 
-        // TODO: Are following even needed? Seems like it throws on error already
-        if (!result.status) {
-            throw new Error('Request failed');
-        }
-        const status: any = { ...result.status };
-        if (!status.SuccessValue || typeof status.SuccessValue !== 'string') {
+        if (!result.status || typeof result.status !== 'object' || !result.status.SuccessValue || result.status.SuccessValue !== 'string') {
```

Comment:
TS didn't like the spread here so I went ahead and condensed this a bit (bonus `TODO` removal 😎)